src/core/public/core_system.test.ts

```diff
@@ -429,15 +429,14 @@ describe('Notifications targetDomElement', () => {
       rootDomElement,
     });
 
-    let targetDomElementParentInStart: HTMLElement | null;
+    let targetDomElementInStart: HTMLElement | null;
     MockNotificationsService.start.mockImplementation(({ targetDomElement }): any => {
-      expect(targetDomElement.parentElement).not.toBeNull();
-      targetDomElementParentInStart = targetDomElement.parentElement;
```

Comment:
AFAIK it does not actually matter that the DOM element passed to Notifications is attached to the root element when `NotificationsService#start` is called, as long as it is attached at some point.

In the future, this pattern should probably be replaced by the single React-tree pattern provided by RenderingService that ChromeService and ApplicationService leverage.